sycl/test-e2e/AddressSanitizer/common/kernel-filter.cpp

```diff
@@ -0,0 +1,27 @@
+// REQUIRES: linux
+// RUN: %{build} %device_asan_flags -O2 -fsanitize-ignorelist=%p/ignorelist.txt -o %t
+// RUN: %{run} not %t &> %t.txt ; FileCheck --input-file %t.txt %s
```

Comment:
can we just pipe the input with like

```
 %{run} not %t &> | FileCheck --input-file %s
```

or does that not work

Reply:
sure, it can